src/nvme/types.h

```diff
@@ -1743,7 +1743,8 @@ struct nvme_id_ns {
 	__u8			nguid[16];
 	__u8			eui64[8];
 	struct nvme_lbaf	lbaf[64];
-	__u8			vs[3712];
+	__le64			lbstm;
```

Comment:
Please also update the documentation accordingly.

Reply:
Thanks for the catch, updated.